vars/configMap.groovy

```diff
@@ -14,7 +14,6 @@ def call(Map input) {
 }
 
 def call(ConfigMapInput input) {
-    assert input.projectName?.trim() : "Param projectName should be defined."
     assert input.configMapName?.trim()  : "Param configMapName should be defined."
 
     echo "Read ConfigMap: ${input.projectName}/${input.configMapName}"
```

Comment:
Can we either remove this line or change it such that we don't echo `Read ConfigMap: /<configMap>` when `projectName is empty?

Reply:
Moved it into the DSL block, so it always prints out the project which the DSL has selected